UC-Berkeley-Extension/global
# Removes modules and updates site configuration.

Removes three modules that we will not normally use.

To test:

* Checkout this branch
* Build the site
  * From inside the VM, run `/var/www/html/scripts/setup.sh`
* Check that comment, history, and quickedit modules are disabled

This PR also sets the default theme to Bartik, since the `bglobal` theme throws numerous errors. It also move the `development.services.yml` file to a config install.
